modules/partition-replica/src/main/java/org/apache/ignite/internal/partition/replica/network/PartitionReplicationMessageGroup.java

```diff
@@ -15,52 +15,51 @@
+import org.apache.ignite.internal.partition.replica.network.disaster.LocalPartitionStatesRequest;
+import org.apache.ignite.internal.partition.replica.network.disaster.LocalPartitionStatesResponse;
+import org.apache.ignite.internal.partition.replica.network.message.HasDataRequest;
+import org.apache.ignite.internal.partition.replica.network.message.HasDataResponse;
+import org.apache.ignite.internal.partition.replica.network.raft.SnapshotMetaRequest;
+import org.apache.ignite.internal.partition.replica.network.raft.SnapshotMetaResponse;
+import org.apache.ignite.internal.partition.replica.network.raft.SnapshotMvDataRequest;
+import org.apache.ignite.internal.partition.replica.network.raft.SnapshotMvDataResponse;
+import org.apache.ignite.internal.partition.replica.network.raft.SnapshotMvDataResponse.ResponseEntry;
+import org.apache.ignite.internal.partition.replica.network.raft.SnapshotTxDataRequest;
+import org.apache.ignite.internal.partition.replica.network.raft.SnapshotTxDataResponse;
+import org.apache.ignite.internal.partition.replica.network.replication.BinaryRowMessage;
+import org.apache.ignite.internal.partition.replica.network.replication.BinaryTupleMessage;
+import org.apache.ignite.internal.partition.replica.network.replication.BuildIndexReplicaRequest;
+import org.apache.ignite.internal.partition.replica.network.replication.ReadOnlyDirectMultiRowReplicaRequest;
+import org.apache.ignite.internal.partition.replica.network.replication.ReadOnlyDirectSingleRowReplicaRequest;
+import org.apache.ignite.internal.partition.replica.network.replication.ReadOnlyMultiRowPkReplicaRequest;
+import org.apache.ignite.internal.partition.replica.network.replication.ReadOnlyScanRetrieveBatchReplicaRequest;
+import org.apache.ignite.internal.partition.replica.network.replication.ReadOnlySingleRowPkReplicaRequest;
+import org.apache.ignite.internal.partition.replica.network.replication.ReadWriteMultiRowPkReplicaRequest;
+import org.apache.ignite.internal.partition.replica.network.replication.ReadWriteMultiRowReplicaRequest;
+import org.apache.ignite.internal.partition.replica.network.replication.ReadWriteScanRetrieveBatchReplicaRequest;
+import org.apache.ignite.internal.partition.replica.network.replication.ReadWriteSingleRowPkReplicaRequest;
+import org.apache.ignite.internal.partition.replica.network.replication.ReadWriteSingleRowReplicaRequest;
+import org.apache.ignite.internal.partition.replica.network.replication.ReadWriteSwapRowReplicaRequest;
+import org.apache.ignite.internal.partition.replica.network.replication.ScanCloseReplicaRequest;
 
 /**
  * Message group for the table module.
  */
-@MessageGroup(groupType = GROUP_TYPE, groupName = "TableMessages")
-public interface TableMessageGroup {
+@MessageGroup(groupType = GROUP_TYPE, groupName = "PartitionReplicationMessages")
+public interface PartitionReplicationMessageGroup {
     /** Table message group type. */
     short GROUP_TYPE = 9;
```

Comment:
There are no messages in table module any longer? Is it true for GG9?

Reply:
In GG9 we have also
`/**
     * Message type for {@link ContinuousQueryScanRequest}.
     */
    short CONTINUOUS_QUERY_SCAN_REQUEST = 24;

    /**
     * Message type for {@link TimedBinaryRowMessage}.
     */
    short TIMED_BINARY_ROW_MESSAGE = 25;

    /**
     * Message type for {@link ReadOnlyStorageOperationReplicaRequest}.
     */
    short RO_STORAGE_OPERATION_REPLICA_REQUEST = Short.MAX_VALUE;`

Reply:
In the table module (ignite-3) - no

Reply:
So, seems that we will have non-trivial merge.